src/SprykerSdk/Zed/ComposerConstrainer/ComposerConstrainerConfig.php

```diff
@@ -44,6 +44,18 @@ public function getCoreNamespaces(): array
         return $this->get(ComposerConstrainerConstants::CORE_NAMESPACES);
     }
 
+    /**
+     * @api
+     *
+     * @return string[]
+     */
+    public function getStrictValidationIgnoredPackages(): array
+    {
+        return [
+            'spryker/kernel$', # kernel needs different evaluation
```

Comment:
The closing $ is a bit weird, can we keep the string non-regex related and just use that character inside the final usage?